livechat/utils/httpx_logger.py

```diff
@@ -37,8 +40,12 @@ def log_response(self, response: httpx.Response) -> None:
         ''' Logs response details. '''
         if not self.disable_logging:
             response.read()
+            try:
+                response_content = json.dumps(response.json(), indent=4)
+            except ValueError:
+                response_content = response.text  # to avoid error when response contains binary data
```

Comment:
```suggestion
                response_content = response.text[:1000]  # to avoid error when response contains binary data
```
or response.text[:[:LOG_CONTENT_LIMIT]]